dipy/workflows/viz.py

```diff
@@ -69,19 +69,17 @@ def run(self, input_files, cluster=False, cluster_thr=15.,
         bg_color : variable float, optional
             Define the background color of the scene. Colors can be defined
             with 1 or 3 values and should be between [0-1].
-            Default is black (e.g --bg_color 0 0 0 or --bg_color 0).
         disable_order_transparency : bool, optional
-            Default False. Use depth peeling to sort transparent objects.
+            Use depth peeling to sort transparent objects.
             If True also enables anti-aliasing.
         buan : bool, optional
-            Enables BUAN framework visualization. Default is False.
+            Enables BUAN framework visualization.
         buan_thr : float, optional
-            Default 0.5. Uses the threshold value to highlight segments on the
+            Uses the threshold value to highlight segments on the
             bundle which have pvalues less than this threshold.
         buan_highlight : variable float, optional
             Define the bundle highlight area color. Colors can be defined
             with 1 or 3 values and should be between [0-1].
-            Default is red (e.g --buan_highlight 1 0 0)
```

Comment:
For example, a value of (1, 0, 0) would mean the red color.

Reply:
Addressed in ce1f6b1